braket_features/Getting a device and checking device properties.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting a device and checking device properties\n",
```

Comment:
I'd probably use "Retrieving devices" instead of "Getting a device".

Reply:
I still prefer to use 'get', as 'retrieve' gives the feeling that we left something and take it back.

Reply:
"Getting devices" then. I'll point out the existing documentation already uses "retrieve": https://docs.aws.amazon.com/braket/latest/developerguide/braket-using-boto3-client.html